imap_processing/run_processing.py

```diff
@@ -0,0 +1,122 @@
+"""Serves as a command line utility to run the processing for a specific
```

Comment:
One thing I was thinking was that we may need to add something like below code block to be able to call from batch command. I don't remember the exact reason, @bryan-harter may know more about it. EMM does something like this and it has something to do with `Entrypoint` or something.
```
#!/usr/bin/env python3
```

Reply:
That is if you want to run it without specifying `python xyz.py` because it will look up your current python3 virtual environment for you and use that.

Reply:
I see. So if we specify exact python comamnd with python file with its argument, then we don't need to add that `#!/usr/bin/env python3` in the code. Is that right?

Reply:
Correct. It is still fine to add that line though if someone does want to execute it as a script, so you can do both.

Reply:
Good to know. Thank you!